content-services/latest/install/ansible.md

```diff
@@ -24,16 +24,45 @@ There are two types of installations - local and remote:
 
 ## Prerequisites
 
-If you're using the Content Services (Enterprise), then you need credentials to access the necessary artifacts from [Nexus](https://artifacts.alfresco.com/nexus/){:target="_blank"}. Customers can request these through [Hyland Community](https://community.hyland.com/){:target="_blank"}.
+If you're using the Content Services (Enterprise), then you need credentials to access the necessary artifacts from 
+[Nexus](https://artifacts.alfresco.com/nexus/){:target="_blank"}. Customers can request these through 
+[Hyland Community](https://community.hyland.com/){:target="_blank"}.
 
 ## Target O/S
-
-The playbooks have been tested using Ansible 2.9.21 (or later) on target hosts with the following operating systems:
-
-* CentOS 7 and 8
-* Red Hat Enterprise Linux 7 and 8
-
-Additional target environments will be added in future releases.
+While Content Services supports a wide range of OS, the playbook is only supported for a subset of them. The table below 
+gives detailed information on the status of supported OS (Additional target environments will be added in future releases):
+
+| OS Flavor / version | 7.x Enterprise | 6.2.2 Enterprise | Community |
+|-|-|-|-|
+| RHEL 8.2 | [x] | - | [x] |
+| RHEL 7.7 | [x] | - | [x] |
+| RHEL 7.6 | [x] | [x] | [x] |
+| CentOS 8 x64 | [x] | - | [x] |
+| CentOS 7 x64 | [x] | [x] | [x] |
+| Ubuntu 20.04 | [x] | - | [x] |
+| Ubuntu 18.04 | [x] | [x] | [x] |
+
+>**Note:** Ansible version 2.12.x is used for testing this playbook.
+
+## Component versions
+The table below shows the version of the components deployed by the playbook for ACS 7.x and 6.2.N.
+
+| Component | 7.1 Enterprise | 7.0.N Enterprise | 6.2.N Enterprise | Community |
+|-|-|-|-|-|
+| OpenJDK | 11.0.13 | 11.0.13 | 11.0.13 | 11.0.13 |
+| Apache Tomcat | 9.0.54 | 9.0.54 | 8.5.72 | 9.0.54 |
+| PostgreSQL | 13.x | 13.x | 11.x | 13.x |
+| Apache ActiveMQ | 5.16.1 | 5.16.1 | 5.15.14 | 5.16.1 |
+| Repository | 7.1.1 | 7.0.1.4 | 6.2.2 | 7.1.1.2 |
+| Share | 7.1.1 | 7.0.1.4 | 6.2.2 | 7.1.1.2 |
+| Search Services | 2.0.2 | 2.0.1.1 | 1.4.3 | 2.0.2 |
+| All-In-One Transformation Engine | 2.5.6 | 2.3.10 | 2.5.6 | 2.5.6 |
+| AOS | 1.4.0 | 1.4.0 | 1.3.1 |  |
+| GoogleDocs | 3.2.1 | 3.2.1 | 3.2.0 |  |
+| Digital Workspace | 2.6.0 | 2.1.0 | 2.6.0 | N/A |
+| Transform Router | 1.5.1 | 1.3.2 | 1.5.1 | N/A |
+| Shared File Store | 0.16.1 | 0.13.0 | 0.16.1 | N/A |
+| Sync Service | 3.5.0 | 3.4.0 | 3.3.3.1 | N/A |
 
```

Comment:
Will need 7.2 column (not ready yet see OPSEXP-1350)

Reply:
This branch does not cover Ansible installation, as pointed out in the name of the PR